dockerfiles/piraeus-server/entry.sh

```diff
@@ -7,6 +7,7 @@ try_import_key() {
   destcrtstore=$3
   tmpfile=$(mktemp)
 
+  rm -f "$destkeystore" "$destcrtstore"
   openssl pkcs12 -export -in "${indir}/tls.crt" -inkey "${indir}/tls.key" -out "$tmpfile" -name linstor -passin 'pass:linstor' -passout 'pass:linstor'
   keytool -importkeystore -srcstorepass linstor -deststorepass linstor -keypass linstor -srckeystore "$tmpfile" -destkeystore "$destkeystore"
```

Comment:
It might be a better idea to include `-noprompt` here, too? From my reading of the manpage this would also solve the issue, and might be less destructive. Altough I can't really see a scenario were someone just wants to partially update the keystore....

Reply:
Hmm, it does seem to import the keystore, but not the certificate:

```
Importing keystore /tmp/tmp.7C2lid7JN1 to /tmp/keystore.jks...
Warning: Overwriting existing alias linstor in destination keystore
Entry for alias linstor successfully imported.
Import command completed:  1 entries successfully imported, 0 entries failed or cancelled

Warning:
The JKS keystore uses a proprietary format. It is recommended to migrate to PKCS12 which is an industry standard format using "keytool -importkeystore -srckeystore /tmp/keystore.jks -destkeystore /tmp/keystore.jks -deststoretype pkcs12".
keytool error: java.lang.Exception: Certificate not imported, alias <ca> already exists
```

So perhaps `rm` is still the safer option